package.json

```diff
@@ -16,5 +17,8 @@
     "start-backend": "cd scripts && \"./startBackendWindows1.sh\"",
     "finish-backend": "cd scripts && \"./startBackendWindows2.sh\"",
     "frontend": "cd scripts && \"./startFrontend.sh\""
+  },
+  "dependencies": {
+    "core-js": "^3.6.5"
```

Comment:
same with this one as well:
remove this dependency, since ur not using it anywhere I think, 
**npm install** inside **root** folder,  and try again if it builds properly